remove unused function with_layout_node #4572

kartaa opened the pull request:
Update wrapper.rs for Issue #4572 in servo/servo.

highfive commented:
Thanks for the pull request, and welcome! The Servo team is excited to review your changes, and you should hear from @jdm (or someone else) soon.

jdm commented:
Great! I've left one comment on Critic that will need to be addressed; make sure to click the 'sign in' link so you receive further updates by email.

kartaa commented:
Done with the change. I was confused if the comment belonged to the deleted function or the class implementation.

Also, apart from this, can you point me to the piece of documentation that could help me understand the meaning of ' in the following code.
```
impl<'ln> LayoutNode<'ln> {
```

Ms2ger commented:
Thanks! If you squash the two commits together, we'll merge them.

There's some documentation at <http://doc.rust-lang.org/book/ownership.html#lifetimes>.

kartaa committed:
remove unused function with_layout_node #4572

remove documentation in with_layout_node #4572

remove unused documentation in function with_layout_node #4572
